src/components/enter-password/enter-password-routes.ts

```diff
@@ -4,25 +4,25 @@ import {
   enterPasswordPost,
   enterPasswordGet,
 } from "./enter-password-controller";
-import { asyncHandler } from "../../utils/async";
 import { validateEnterPasswordRequest } from "./enter-password-validation";
 import { requiresAuthMiddleware } from "../../middleware/requires-auth-middleware";
 import { refreshTokenMiddleware } from "../../middleware/refresh-token-middleware";
+import { globalTryCatch } from "../../utils/global-try-catch";
 
 const router = express.Router();
 
 router.get(
   PATH_DATA.ENTER_PASSWORD.url,
   requiresAuthMiddleware,
-  enterPasswordGet
+  globalTryCatch(enterPasswordGet)
 );
 
 router.post(
   PATH_DATA.ENTER_PASSWORD.url,
   requiresAuthMiddleware,
   validateEnterPasswordRequest(),
   refreshTokenMiddleware(),
-  asyncHandler(enterPasswordPost())
+  globalTryCatch(enterPasswordPost())
```

Comment:
should this be async?

Reply:
Thanks for spotting, this has been changed to use correct global try/catch and async handler